### PR TITLE
TT-119: Switching to a different committee doesn't do anything (FE)

### PR DESCRIPTION
**Pull request by elijah-parrish**

### CODE REVIEW

File: src/components/CommitteesMenu.vue [hunk 1]
```diff
@@ -74,6 +82,19 @@ export default {
...
+  border: none;
+  padding: 0!important;
+  text-transform: uppercase;
+  font-size: 1.5em;
+  /*optional*/
+  font-family: 'Montserrat', Helvetica, Arial, sans-serif;
+  /*input has OS specific font-family*/
+  color: rgb(0,0,0);
+  cursor: pointer;
+}
```
Comment (omardlhz) [inline src/components/CommitteesMenu.vue]: Please fix CSS indentation

File: src/components/CommitteesMenu.vue [hunk 2]
```diff
@@ -27,7 +27,8 @@ export default {
   data () {
     return {
       committees: null,
-      loading: true
+      loading: true,
+      url: ''
```
Comment (omardlhz) [inline src/components/CommitteesMenu.vue]: Is this url being used?

File: src/components/CommitteesMenu.vue (hunk 1 above)
Comment (elijah-parrish) [reply src/components/CommitteesMenu.vue]: Done!

File: src/components/CommitteesMenu.vue (hunk 2 above)
Comment (elijah-parrish) [reply src/components/CommitteesMenu.vue]: Must be dead code from beforehand. I removed it

**Pull request merged**